### PR TITLE
Pass: Chromium Frame.GoTo tests

### PR DESCRIPTION
**Pull request by kblok**
relates to #70

### CODE REVIEW

File: src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs [hunk 1]
```diff
@@ -23,5 +40,188 @@ internal Task SetUserAgentAsync(string userAgent)
...
+                System.Diagnostics.Debug.WriteLine(ex.Message);
+                _client.OnClosed(ex.Message);
+            }
+        }
+
+        private void OnLoadingFinished(NetworkLoadingFinishedChromiumEvent e)
+        {
+            // For certain requestIds we never receive requestWillBeSent event.
+            // @see https://crbug.com/750469
+            if (!_requestIdToRequest.TryGetValue(e.RequestId, out var request))
```
Comment (Meir017) [inline src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs:86]: consider using `TryRemove` and delete the `_requestIdToRequest.Remove(request.RequestId);` call

File: src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs [hunk 2]
```diff
@@ -23,5 +40,188 @@ internal Task SetUserAgentAsync(string userAgent)
...
+
+        private void OnRequest(NetworkRequestWillBeSentChromiumEvent e, string interceptionId)
+        {
+            if (e.Request.Url.StartsWith("data:"))
+            {
+                return;
+            }
+
+            var redirectChain = Array.Empty<Request>();
+            /*TODO
```
Comment (Meir017) [inline src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs:116]: todo?

File: src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs [hunk 3]
```diff
@@ -23,5 +40,188 @@ internal Task SetUserAgentAsync(string userAgent)
...
+                }
+            }
+            }*/
+
+            if (!_page.FrameManager.Frames.TryGetValue(e.FrameId, out var frame))
+            {
+                return;
+            }
+
+            bool isNavigationRequest = e.RequestId == e.LoaderId && e.Type == Protocol.Network.ResourceType.Document;
```
Comment (Meir017) [inline src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs:134]: why not add using for this?

File: src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs (hunk 2 above)
Comment (kblok) [reply src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs:116]: Scope. I will solve that when I solve Redirect tests.

File: src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs (hunk 3 above)
Comment (kblok) [reply src/PlaywrightSharp.Chromium/ChromiumNetworkManager.cs:134]: Conflicts with `PlaywrightSharp.ResourceType`.

**Pull request merged**